README.md

```diff
@@ -10,6 +10,7 @@ java -jar ./shopping/target/shopping-0.9.0-SNAPSHOT.jar server ./shopping/app.ya
 java -jar ./styling/target/styling-0.9.0-SNAPSHOT.jar server ./styling/app.yaml
 java -jar ./printing/target/printing-0.9.0-SNAPSHOT.jar ./printing/app.yaml
 java -jar ./packaging/target/packaging-0.9.0-SNAPSHOT.jar ./packaging/app.yaml
+java -jar ./delivery/target/delivery-0.9.0-SNAPSHOT.jar
```

Comment:
specify config file so that anyone who wants customize will know.
```suggestion
java -jar ./delivery/target/delivery-0.9.0-SNAPSHOT.jar  --spring.config.location=./delivery/src/main/resources/application.yml 
```